tests/fiaas_deploy_daemon/deployer/kubernetes/test_deployment_deploy.py

```diff
@@ -81,12 +86,14 @@ def environment(self, request):
         yield request.param
 
     @pytest.fixture(params=(
-        # key: (infrastructure, global_env, use_in_memory_emptydirs, disable_deprecated_managed_env_vars)
-        ("gke", {}, True, True),
+        # key: (infrastructure, global_env, set[Feature]
+        ("gke", {}, {Feature.USE_IN_MEMORY_EMPTYDIRS, Feature.DISABLE_DEPRECATED_MANAGED_ENV_VARS}),
+        ("gke-w/service-per-acct", {}, {Feature.USE_IN_MEMORY_EMPTYDIRS, Feature.DISABLE_DEPRECATED_MANAGED_ENV_VARS,
+         Feature.ENABLE_SERVICE_ACCOUNT_PER_APP}),
         ("diy", {
             'A_GLOBAL_DIGIT': '0.01',
             'A_GLOBAL_STRING': 'test',
-        }, True, True),
+        }, {Feature.USE_IN_MEMORY_EMPTYDIRS, Feature.DISABLE_DEPRECATED_MANAGED_ENV_VARS}),
```

Comment:
This is a really nice improvement for the testing 🏆

Reply:
Kudos to @stribb 🥇